### PR TITLE
#502 Fix reload after login

### PR DESCRIPTION
**Pull request by fkroockmann**
https://github.com/backbee/BbCoreJs/issues/502

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
if it works, it's so cool <3

File: src/tb/apps/main/main.js [hunk 1]
```diff
@@ -89,7 +89,9 @@ define(
                 if (current_url !== null) {
                     Core.RouteManager.navigateByPath(current_url);
                 } else {
-                    Core.RouteManager.navigateByPath(Core.config('default_url'));
+                    if (Core.get('redirect_default_url') !== false) {
```
Comment (ndufreche) [inline src/tb/apps/main/main.js]: I don't understand redirect_default_url is all time false or undefined

Comment (fkroockmann) [inline src/tb/apps/main/main.js]: No, redirect_default_url is false when on:success:login is trigged.
Cause on:succes:login, the application is init and main Application is init too

Comment (ndufreche) [inline src/tb/apps/main/main.js]: Ok

**Comment by ndufreche** [comment]
:+1: thanks

**Pull request merged**